### PR TITLE
First run at fixing WP JS Coding Standards

### LINKED ISSUE #822: WordPress JavaScript Coding Standards need to be fixed
There are numerous places within `index.js` that do not meet the WordPress JavaScript Coding Standards.

### PR DESCRIPTION
**Pull request by ianbelanger79**
I think I got all of the JS CS violations, but would love a review of someone more well versed in the JS Coding Standards

Fixes #822

### CODE REVIEW

**Comment by ntwb** [comment]
I’ll review this in more detail later today, to also ensure it passes WP Core JSHint checks too

File: assets/js/index.js [hunk 1]
```diff
@@ -93,10 +93,8 @@ twentytwenty.coverModals = {
 	// Handle cover modals when they're toggled
 	onToggle: function() {
 		document.querySelector( '.cover-modal' ).addEventListener( 'toggled', function( event ) {
-			var modal, body;
-
-			modal = event.target;
-			body = document.body;
+			var modal = event.target,
+				body = document.body;
```
Comment (acosmin) [inline assets/js/index.js]: The indentation looks off in this situations :)

Comment (acosmin) [reply assets/js/index.js]: Also, sorry for coding them like this, that's how the previous linter wanted them...

Comment (ianbelanger79) [reply assets/js/index.js]: The indentation is required by the handbook. https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#declaring-variables-with-var

**Comment by ianbelanger79** [comment]
Closing because of updated PR here #938

**Pull request closed without merging**